### PR TITLE
feat: Ability to add list tags

### PR DESCRIPTION
**Pull request by AyakaOnozuka131**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const prentElement = document.querySelector("ul");
+const childElement = document.createElement("li");
+childElement.innerText = "これです";
```
Comment (kenmori) [inline index.js]: Q
why did you use innerText instead of textContent

File: index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+const prentElement = document.querySelector("ul");
```
Comment (kenmori) [inline index.js]: Q
why did you use querySelector instead of getElementById

File: index.js (hunk 1 above)
Comment (AyakaOnozuka131) [reply index.js]: Thanks for the review!
As you said, this time it was just simple text, so we will change it to textContent.

File: index.js (hunk 2 above)
Comment (AyakaOnozuka131) [reply index.js]: Thanks for the review!
Now that I know I can add the ID attribute, I will add it and fix the problem.

**Comment by AyakaOnozuka131** [comment]
# [Issue No.1](https://github.com/kenmori/handsonFrontend/blob/0087a459ae7e84af59ead2d97d9c6b5153481e69/work/markup/1.md#dom%E6%A7%8B%E7%AF%89)

## [CodeSandBox](https://codesandbox.io/s/javascript-lessons-01-e07hk?file=/index.js)

## latest commit : [8a2360f](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/1/commits/8a2360f6d93879746d0050d88e2b26883a1390fe)

## Concerns and areas to focus on

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# [Issue No.](リンク)
+
+**課題内容**
+
+## [CodeSandBox](リンク) :最新のコミットリンク。SandBox内の内容と最新のコミットリンクまでの内容を合わせる。合っていない場合レビュワーは拒否できます
+
+## 実装上、不安なところ、重点的に見てもらいたいところ、聞きたいこと
+- コミットリンク
+- コミットリンク
```
Comment (kenmori) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: 👍

**Review by kenmori — APPROVED** [review]

**Pull request merged**